src/main/java/org/cactoos/iterable/IterableEnvelope.java

```diff
@@ -39,66 +32,39 @@
  *
  * @param <X> Type of item
  * @since 0.24
- * @checkstyle AbstractClassNameCheck (500 lines)
  */
-@SuppressWarnings("PMD.AbstractNaming")
 public abstract class IterableEnvelope<X> implements Iterable<X> {
 
     /**
      * The iterable.
      */
-    private final Unchecked<Iterable<X>> iterable;
+    private final Iterable<X> wrapped;
 
     /**
      * Ctor.
-     * @param scalar The source
+     * @param wrapped The wrapped iterable
      */
-    public IterableEnvelope(final Scalar<Iterable<X>> scalar) {
-        this.iterable = new Unchecked<>(scalar);
+    public IterableEnvelope(final Iterable<X> wrapped) {
```

Comment:
@victornoel The parameter's name and attribute must be different. Please, can you change it?

Reply:
@fabriciofx why should they be different? I'm not clear on the rational behind that

Reply:
@victornoel it's a bad practice because is harder than read (different name is easier). Who said it? Yegor and if you pay attention, you'll see these names are (or should be) different ia cactoos' classes.

Reply:
@fabriciofx thx, didn't know that, I will change them, the argument is convincing :) btw do you known if there is a blog post about that matter somewhere, I would be interested on reading about it more!

Reply:
@victornoel I don't remember if there is a blog post about it, but once I pushed a PR and Yegor alert me about it...